lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -15,8 +13,6 @@ module Actions
     class FirebaseAppDistributionAction < Action
       DEFAULT_FIREBASE_CLI_PATH = `which firebase`
```

Comment:
This isn't used anymore so feel free to remove it.

Reply:
Can we remove the  configitem?

https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/blob/bb0dfea89fb4cb62ce8e63722ae98afc4234280c/lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb#L102-L118

Reply:
Ahh good point. I think we can remove this _if_ it doesn't break existing customers that have the `firebase_cli_path` set. We will have to test this scenario, and I'm not sure this is really in the scope of this PR so I'd so maybe hold off on this change 🙂